### PR TITLE
fix(async-csv): Handle unicode in log messages

### PR DESCRIPTION
**Pull request by Zylphrex**

### CODE REVIEW

**Comment by Zylphrex** [comment]
It may be easier to review if you hide the white space changes.

File: src/sentry/data_export/utils.py [hunk 1]
```diff
@@ -10,46 +10,64 @@
...
+                    "dataexport.error", tags={"error": six.text_type(error)}, sample_rate=1.0
+                )
+                logger.info("dataexport.error: %s", six.text_type(error))
+                capture_exception(error)
+                raise ExportError("Invalid date range. Please try a more recent date range.")
+            except snuba.QueryIllegalTypeOfArgument as error:
+                metrics.incr(
+                    "dataexport.error", tags={"error": six.text_type(error)}, sample_rate=1.0
+                )
+                logger.info("dataexport.error: %s", six.text_type(error))
```
Comment (markstory) [inline src/sentry/data_export/utils.py:38]: You could skip the logs as you're capturing exceptions, which will give you way better information.

**Review by markstory — APPROVED** [review]

File: src/sentry/data_export/utils.py (hunk 1 above)
Comment (Zylphrex) [reply src/sentry/data_export/utils.py:38]: These logs are quite useful in tests where (correct me if I'm wrong) there is no way to see the exception in sentry.

Comment (markstory) [reply src/sentry/data_export/utils.py:38]: Yes that is true.

**Pull request merged**